packages/osl-operator-bundle-image/resources/scripts/bump_csv.py

```diff
@@ -33,9 +33,9 @@ def update_versioning_csv(new_version: str) -> None:
     with CSV_BASE_PATH.open("r") as f:
         data = yaml.load(f)
 
-    data["metadata"]["name"] = f"logic-operator-rhel8.v{new_version}"
+    data["metadata"]["name"] = f"logic-operator-rhel9.v{new_version}"
     data["spec"]["version"] = new_version
-    data["spec"]["replaces"] = f"logic-operator-rhel8.v{previous_version(new_version)}"
+    data["spec"]["replaces"] = f"logic-operator-rhel9.v{previous_version(new_version)}"
```

Comment:
This csv bump for bundle image might be specific in case of next release. Because the version bump will be from rhel8 image to rhel9 image.
Right now we should keep in replace the rhel8 and after next release update it to new image

Reply:
Not sure I follow, could you elaborate? New files will have rhel9 prefix.

Reply:
yes, i think here it requires to be the previous released OLM so rhel-8 should be kept for this release

Reply:
```suggestion
    data["spec"]["replaces"] = f"logic-operator-rhel8.v{previous_version(new_version)}"
```
proposed change to keep rhel8 there